views/pages/sqlInjection.gohtml

```diff
@@ -1,8 +1,19 @@
 {{define "sqlInjection"}}
-<h1 class="page-header">{{.Name}}</h1>
-{{template "ruleInfo" .}}
-{{$base := .Base}}
-{{range .Sinks}}
+{{$routeInfo := index .Rulebar .Name}}
+{{$base := $routeInfo.Base}}
+{{$port := .Port}}
+{{$inputs := $routeInfo.Inputs}}
+<h1 class="page-header">{{$routeInfo.Name}}</h1>
+{{template "ruleInfo" $routeInfo}}
+<div class="row">
+  <div class="col-xs-12 col-sm-6" style="padding-bottom: 30px;">
+    <h4 class="sub-header">Headers - JSON</h4>
+    <p><pre>curl http://localhost{{$port}}{{$base}}/{{index $inputs 1}}/json/unsafe -X POST -H "Content-Type: application/json" -H "credentials:{\"username\":\"Robert'; DROP TABLE Students;--\",\"password\":\"12345Pass\"}"</pre></p>
+    <p><pre>curl http://localhost{{$port}}{{$base}}/{{index $inputs 1}}/json/safe -X POST -H "Content-Type: application/json" -H "credentials:{\"username\":\"Robert'; DROP TABLE Students;--\",\"password\":\"12345Pass\"}"</pre></p>
+    <p><pre>curl http://localhost{{$port}}{{$base}}/{{index $inputs 1}}/json/noop -X POST -H "Content-Type: application/json" -H "credentials:{\"username\":\"Robert'; DROP TABLE Students;--\",\"password\":\"12345Pass\"}"</pre></p>
+  </div>
+</div>
+{{range $routeInfo.Sinks}}
 <div class="row">
   <div class="col-xs-12 col-sm-6" style="padding-bottom: 30px;">
     <h4 class="sub-header">
```

Comment:
I think this `<h4 class="sub-header">` block should be added to the row above as well. Both use the same vulnerable sink so it should be referenced in both places. 

Alternately: the path traversal page puts this header at the top of the entire block, so this one could be moved towards the top of the page and just appear once.

Reply:
You're absolutely right, it's kinda misleading. I'll update it so `sqlit3.exec` holds the other two as sources for the same sink, like in Path Traversal.